### ISSUE TITLE
Harvest ArcGIS Server into AGOL/Portal extent issues

### ISSUE THREAD

**Issue by nmpdwr** [Esri/geoportal-server-harvester#48]
Hello,
   I am running into an issue where the harvest into AGOL and Portal for ArcGIS Server have the extents improperly populated.  When I manually register a map service the extent is correctly set. Harvesting however seems to be setting the values in the original meters as listed in the service and not converting the number to Decimal degrees.
   Is this a known issue or is there something i may be missing in terms of configuration?

Thank you
nmp

**Comment by pandzel** [comment]
I understand that you are harvesting content of the ArcGIS Server into a the Portal (or AGOL), is that so? I this server available publicly? Can you provide an URL into it if possible with indication which exactly map service is giving the problems?

**Comment by nmpdwr** [comment]
Yes, the service endpoint and the harvested entities are publicly available.  This is happening for all the services harvested, none get the proper extent. I can share the Esri open data entry, since our Organization AGOL site does not allow direct access without authentication
Example:
Feature service harvested into AGOL
https://gis.water.ca.gov/arcgis/rest/services/Boundaries/i03_WaterDistricts/FeatureServer

Open Data site entry, with incorrect extent
https://atlas-dwr.opendata.arcgis.com/datasets/45d26a15b96346f1816d8fe187f8570d_0
Here is the metadata entry page
https://www.arcgis.com/home/item.html?id=45d26a15b96346f1816d8fe187f8570d


The extent that was populated upon harvest is:
Top: 5,201,773.72 Left: -13,838,799.71 Right: -12,747,459.36 Bottom: 3,833,610.93

**Comment by pandzel** [comment]
Pull request #50 solves the issue.
